extensions/wikia/CommunityPage/helpers/LinkHelper.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+class LinkHelper {
+	public static function forceLoginLink( Title $title, $edit=true ) {
+		global $wgUser, $wgDisableAnonymousEditing;
+
+		if ( $wgDisableAnonymousEditing && !$wgUser->isLoggedIn() ) {
+			return SpecialPage::getTitleFor( 'SignUp' )->getFullURL(
+				[
+					'returnto' => $title->getEscapedText(),
+					'returntoquery' => $edit ? urlencode( static::getEditorParam() ) : '',
+					'type' => 'login'
+				]);
+		}
+
+		return $edit ? static::getEditUrl( $title->getFullURL() ) : $title->getFullURL();
+	}
+
+	public static function getEditUrl( $articleUrl ) {
```

Comment:
It seems like this method is used once and it has a single line. Do you think removing it would make the code simpler?

On the other hand, it has `public` modifier. Is it needed?